### PR TITLE
Add small unit testing

### PR DESCRIPTION
**Pull request by peter279k**
# Change log
- Add unit testing for this pacckage.
- Using mock to stub some static methods.
- Using travis-ci to do unit testing. (see [this](https://travis-ci.org/peter279k/ca-bundle))

# Suggestions (If possible)
- It's hard to use PHPUnit to do unit testing for static method. Perhaps using non- static method?

### CODE REVIEW

File: .styleci.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - no_useless_else
+  - ordered_class_elements
+  - ordered_use
+  - strict
+  - strict_param
+  - php_unit_construct
+  - php_unit_strict
+finder:
+  name:
+    - "*.php"
```
Comment (GrahamCampbell) [inline .styleci.yml]: no need for this, this is already the default

Comment (GrahamCampbell) [inline .styleci.yml]: the finder is setup as so:

image

Comment (peter279k) [inline .styleci.yml]: ok, I will remove the ```.styleci.yml```

Comment (GrahamCampbell) [inline .styleci.yml]: what, why?

Comment (GrahamCampbell) [inline .styleci.yml]: i meant just remove the finder config

**Comment by peter279k** [comment]
Hi @GrahamCampbell , I an sorry about I misunderstand what do you mean.
I think using ```.php_cs``` is enough to check the coding style.
So I remove the ```.styleci.yml```. I will add back the ```.styleci.yml``` and remove the ```finder``` config.

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**